.travis.yml

```diff
@@ -3,7 +3,8 @@ language: ruby
 sudo: false
 
 rvm:
-  - jruby
+  - jruby-9.0.5.0
+  - jruby-9.1.9.0
```

Comment:
```
[sfmovies (hwrd-movies-list)]$ rbenv install -l | ag jruby | ag -f '9.1.'
  jruby-9.1.0.0-dev
  jruby-9.1.0.0
  jruby-9.1.1.0
  jruby-9.1.2.0
  jruby-9.1.3.0
  jruby-9.1.4.0
  jruby-9.1.5.0
  jruby-9.1.6.0
  jruby-9.1.7.0
  jruby-9.1.8.0
  jruby-9.1.9.0-dev
  jruby-9.1.9.0
  jruby-9.1.10.0
  jruby-9.1.11.0
  jruby-9.1.12.0
```

There are some newer ones - is there languages that specifies exactly which ones we work with?

Reply:
I think we've traditionally just specified the latest version of jruby. However, there is currently an issue with our TravisCI builds, where we can't access the latest version of jruby, 9.1.13.0 (it turns out this is an issue with jruby, and the thread can be found [here](https://github.com/travis-ci/travis-ci/issues/8446)).

As a temporary fix, I've added the [latest version of jruby which TravisCI provides](http://rubies.travis-ci.org/) to get the builds passing. Once the issue is resolved, I can return this to just be `- jruby`.

Reply:
Makes sense 👍